source/test/project.xml

```diff
@@ -13,6 +13,7 @@
 	
 	<haxelib name="munit" />
 	<haxelib name="flixel" />
+	<haxelib name="flixel-addons" />
```

Comment:
Could you add a line to `.travis.yml` to install `flixel-addons`? I think the right syntax is something like `haxelib install flixel-addons >/dev/null`